src/Aspire.Hosting/Aspire.Hosting.csproj

```diff
@@ -73,6 +73,8 @@
     <InternalsVisibleTo Include="Aspire.Hosting.Testing.Tests" />
     <InternalsVisibleTo Include="Aspire.Hosting.Containers.Tests" />
     <InternalsVisibleTo Include="Aspire.Cli.Tests" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Docker" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Kubernetes" />
```

Comment:
We don't want IVT to product code, only tests. Why do you need this?

Reply:
Its not - sorry, just made it easier for me to play around with the annotations before I'd added the execution context ^^.- will be removing